src/components/UserInfo.js

```diff
@@ -1,19 +1,24 @@
 export default class UserInfo {
-  constructor({ profileName, profileDescription }) {
+  constructor({ profileName, profileDescription, profileAvatar }) {
     this._profileName = document.querySelector(profileName);
     this._profileDescription = document.querySelector(profileDescription);
+    this._avatar = document.querySelector(profileAvatar);
   }
 
   getUserInfo() {
     this._user = {
       name: this._profileName.textContent,
-      description: this._profileDescription.textContent,
+      about: this._profileDescription.textContent,
     };
     return this._user;
   }
 
-  setUserInfo({ name, description }) {
+  setUserInfo({ name, about }) {
     this._profileName.textContent = name;
-    this._profileDescription.textContent = description;
+    this._profileDescription.textContent = about;
+  }
+
+  setAvatar(avatar) {
```

Comment:
It's great to have a separate setter for the avatar as it's controlled by a different form